src/main/java/servlet/Calc.java

```diff
@@ -0,0 +1,117 @@
+package servlet;
+
+import java.util.*;
+import java.util.regex.Pattern;
+
+class Calc {
+    static String replaceVars(Map<String, String> vars) {
+        String equation = vars.remove("expr");
+        for (Map.Entry<String, String> param : vars.entrySet()) {
+            String value;
+            if(Character.isAlphabetic(param.getValue().charAt(0))) { // if variable refers to another variable
+                value = vars.get(param.getValue()); // get value of referral variable
+            } else {
+                value = param.getValue();
+            }
+            equation = equation.replace(param.getKey(), value);
+        }
+        System.out.printf("[+] Replaced vars with their values. {%s}%n", equation);
+        return equation;
+    }
+
+    static int evaluate(String equation) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/thejerome/IFMO_Servlets_20190915/pullRequest?prid=4662868)